load-test-framework/python_src/clients/cps_publisher_task.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+import grpc
+from concurrent import futures
+from google.cloud import pubsub
+
+import loadtest_pb2
+
+
+class LoadtestServicer(loadtest_pb2.LoadtestServicer):
+    """Provides methods that implement functionality of load test server."""
+
+    def __init__(self):
+        self.pubsub_client = pubsub.Client()
```

Comment:
I do if I don't want to recreate it on every check call. This creates stubs internally.